equinoxfitness/datacoco-cloud
# Qa promote to master

## Description

codacy fixes

## Changes

- add updates to athena_interaction from bb
- codacy fixes
- update codacy badges link
- update version

## How Has This Been Tested

1. Checkout this branch
2. Switch to virtual environment
3. Run tox
- [ ] Verify that the test runs successfully
